README.md

```diff
@@ -90,192 +97,113 @@ let y = Named["string name"];
+    auto-incremented by `1` starting from `0` or the last initialized Number value.
+  - If the <var>hint</var> is `"string"`, the default value is the SV of the _enum member_'s name.
+  - If the <var>hint</var> is `"symbol"`, the default value is a `Symbol` whose description is
+    the SV of the _enum member_'s name, prefixed with the SV of the containing _EnumDeclaration_'s 
+    name concatenated with a `.`.
 
 Enum members are `[[Writable]]`: **false**, `[[Enumerable]]`: **false**, and `[[Configurable]]`: 
 **false**.
 
-Enum members can be accessed and modified programmatically via methods on the global `Enum` 
-object, as described in the [API](#API) section.
-
-## Enum Values
-
-Enum values are a new primitive type, `enum`, that are composed of two parts: An _underlying 
-primitive value_ (which must be either a `Number`, `String`, `Symbol`, or `Boolean`) and a 
-reference to its _originating declaration_ (which is used for coercion and equality): 
-
-```js
-typeof Numbers.zero === "enum"
-Numbers.zero.valueOf() === 0
-Numbers.zero.toString() === "zero"
-
-typeof PlayState.idle === "enum"
-PlayState.idle.valueOf() === "idle"
-PlayState.idle.toString() === "idle"
-```
-
-This representation allows multiple use cases:
-- [Strict inequality](#Operators): `Numbers.zero !== Colors.red`
-- [Bitwise operations](#Operators): `Numbers.one | Numbers.two`
-- [Math operations](#Operators): `let state = States.initial; state++;`
-
-### Operators
-ECMAScript operators are supported through existing coercion behavior, with a few minor differences:
-
-- Strict equality (`===`) performs no coercion (no change).
-- Weak equality (`==`) performs coercion for enum values to their _underlying primitive type_.
-- Addition (`+`), Multiplication (`*`), and Binary Bitwise (`&`, `|`) operations perform coercion 
-  for enum values per the existing spec text, however if _both_ operands are enum values whose
-  _underlying primitive values_ are both `Number` and have the same _originating declaration_, the
-  result is coerced back into an enum value with that _originating declaration_.
-  ```js
-  Numbers.one + 2 === 3 // Coerces `Numbers.one` to Number.
-  Numbers.one | Numbers.two === Numbers.three; // Coerces both operands to Number, and the result back to `Numbers`.
-  ```
-- Update (`++`, `--`) and Bitwise NOT (`~`) operations coerce back to the operand's enum type if its _underlying 
-  primitive type_ is `Number`.
-- Bitwise Shift (`<<`, `>>`, `>>>`) operations coerce back to the _left_ operand's enum type if its _underlying
-  primitive type_ is `Number`.
-- Other operations not specified perform coercion as they are specified (no change). 
-
 # API
 
-Enums have the following API:
+To make it easier to work with enums, an `Enum` object is added to the global scope, with the following
+methods:
+
+> NOTE: If `Enum` turns out to be web-incompatible, these methods could instead be added to `Reflect`.
```

Comment:
Reflect is only ever for Proxy traps (I've tried to add things to it before), so I'm quite sure this isn't an option.

Reply:
I'll just remove this for now.